src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_part.c
+ * Overlay: ovl_En_Part
+ * Description: Particle emitter for enemies' death
```

Comment:
I prefer the term "effects" over particles, cause some of the effects do stuff like making sounds. and other things that arent really particles.
I would say Effect Spawner instead of Particle Emitter but thats just me :D